WordPress/twentytwentyfive
# Remove pattern metadata

**Description**
I have removed some left over pattern meta data, which gives us a clean slate to add new names to blocks and sections.

Closes https://github.com/WordPress/twentytwentyfive/issues/402


**Testing Instructions**
Preview or place each pattern that is updated in this PR (see _file changes_) and confirm that there are no block validation errors.
